### DISCUSSION TITLE
Sort incoming frames from Spinnaker.Capture by odd/even frame numbers

### DISCUSSION THREAD

**Discussion by lhaetzel** [bonsai-rx/bonsai#1566] [Q&A] [answered]
I'm using the following workflow to get a online/real-time fluorescence activity readout during a behavioural task. 

Is there any way to sort the incoming camera frames from Spinnaker.Capture so that I end up with 2 separate graphs, one plotting fluorescence measurements from only odd frame numbers and the other from only even frame numbers?

Live graph 1: Frames 1,3,5,7,9....
Live graph 2: Frames 2,4,6,8,10....

Thank you!

![workinversion_fpacq](https://github.com/bonsai-rx/bonsai/assets/52512192/ef0a0da4-5c86-482a-89b9-83205c2a42f7)

**Comment by bruno-f-cruz** [comment, marked as answer]
I think you are looking for the [`Slice` operator](https://bonsai-rx.org/docs/api/Bonsai.Reactive.Slice.html).

**Reply by lhaetzel** [reply]
that works, thanks!
